CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_iterable v0.8.1
```

Comment:
Since this _technically_ impacts a downstream incremental model. Shouldn't this be considered a breaking change? Or would this not actually impact the incremental results?

Reply:
Hmmm I was hesitant since we just cut a breaking change last week for Databricks, but yes, this is a good point as it would impact downstream models for Snowflake and Bigquery. The other warehouses should not be affected. For Snowflake and Bigquery, it would result in rows being added, for example to `iterable__users` and be safer to make a breaking change. I will update and add a comment about a full refresh too.

Reply:
Yeah I have had that hesitation before. There is no downside to cutting another breaking change right after another. This happens quite frequently with packages that have incremental models. Thanks for making the updates!